src/weather_magic/state.cljs

```diff
@@ -25,17 +24,19 @@
                           :month {:value 1 :min 1 :max 12}}))
 
 ;; The function currently animating the earth.
-(defonce earth-animation-fn (atom world/spin))
+(defonce earth-animation-fn (atom nil))
```

Comment:
Yes, vi får ta ett samtal om det här och besluta om vi ska acceptera den här lösningen eller omorganisera vårat tillstånd så att det istället för att ligga i `state.cljs` ligger i de områdesspecifika filerna så som `models.cljs` och `textures.cljs`.

Problemet vi går in i annars är cirkulärberoenden där funktionerna i `world.cljs` modifierar det modifierbara tillståndet i `state.cljs` samtidigt som man i `state.cljs` vill sätta defaultvärden för atomerna från `world.cljs`.